test/specs/reducers/transactions.spec.js

```diff
@@ -7,9 +7,9 @@ describe('Transactions reducer',() =>{
       const action = {
         type: 'GET_TRANSACTIONS_FULFILLED',
         payload: [{ hash: '0x9fc76417374aa880d4449a1f7f31ec597f00b1f6f3dd2d66f4c9c6c445836d8b', 
-                    from: '0xa94f5374fce5edbc8e2a8697c15331677e6ebf0b',
-                    to: '0x6295ee1b4f6dd65047762f924ecd367c17eabf8f',
-                    blockNumber: '3' }],
+          from: '0xa94f5374fce5edbc8e2a8697c15331677e6ebf0b',
+          to: '0x6295ee1b4f6dd65047762f924ecd367c17eabf8f',
+          blockNumber: '3' }],
```

Comment:
The changes here are irrelevant to the issue and should be made in a separate pull request.